code/game/atoms.dm

```diff
@@ -381,6 +381,26 @@
 	if( istype(src, /turf/simulated) )
 		new /obj/effect/decal/cleanable/poop(src)
 
+/atom/proc/addSmell(var/turf/simulated/T, var/smellName, var/concentration)
+	if(istype(T, /turf/simulated))

```

Comment:
You can drop the `, /turf/simulated` part, just `istype(T)` will check if it's a simulated turf as it's casted as such.